File: adminpanel/src/main/java/com/example/adminpanel/MainActivity.java
package com.example.adminpanel;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;
import android.view.MenuItem;

import Fragments.AddFragment;
import Fragments.DonationFragment;
import Fragments.EmergencyFragment;
import Fragments.NeedHelpFragment;
import Fragments.ProjectFragment;

import com.google.android.material.bottomnavigation.BottomNavigationView;




public class MainActivity extends AppCompatActivity {

    private BottomNavigationView bottomNavigationView;
    private Fragment selectorFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNavigationView = findViewById(R.id.bottom_navigation);

        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                switch (item.getItemId()){
                    case R.id.nav_projects:
                        selectorFragment = new ProjectFragment();
                         break;
                    case R.id.nav_donate:
                        selectorFragment = new DonationFragment();
                        break;
                    case R.id.nav_add:
                        selectorFragment = new AddFragment();
                        break;
                    case R.id.nav_emergency:
                        selectorFragment = new EmergencyFragment();
                        break;
                    case R.id.nav_needhelp:
                        selectorFragment = new NeedHelpFragment();
                }
                if (selectorFragment != null){
                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container , selectorFragment).commit();
                }
                return true;
            }
        });

        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container , new ProjectFragment()).commit();
    }
}

File: adminpanel/src/main/java/com/example/adminpanel/MainActivity.kt
package com.example.adminpanel


import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.findNavController
import androidx.navigation.ui.AppBarConfiguration
import androidx.navigation.ui.setupWithNavController
import com.example.adminpanel.databinding.ActivityMainBinding
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    private lateinit var appBarConfiguration: AppBarConfiguration

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        appBarConfiguration = AppBarConfiguration(
            setOf(R.id.navigation_need_help_request,R.id.navigation_emergency,R.id.navigation_donate,R.id.navigation_project,R.id.navigation_add)
        )
        binding.toolbar.setNavigationIconTint(resources.getColor(R.color.green_main))
        binding.toolbar.setBackgroundColor(getColor(R.color.white))
        binding.toolbar.setTitleTextColor(getColor(R.color.green_main))
        setSupportActionBar(binding.toolbar)
//        supportActionBar?.title=""
        val bottomNavView: BottomNavigationView = binding.bottomNavView
        val navController = findNavController(R.id.nav_host_fragment_activity_main)
        bottomNavView.setupWithNavController(navController)
        bottomNavView.setOnItemSelectedListener {
            when (it.itemId) {
                R.id.nav_add -> {
                    navController.navigate(R.id.navigation_add)
                    binding.toolbar.title="Add Post"
                    true
                }
                R.id.nav_donate-> {
                    navController.navigate(R.id.navigation_donate)
                    binding.toolbar.title="Donations"
                    true
                }
                R.id.nav_projects -> {
                    navController.navigate(R.id.navigation_project)
                    binding.toolbar.title="Projects"
                    true
                }
                R.id.nav_emergency -> {
                    navController.navigate(R.id.navigation_emergency)
                    binding.toolbar.title="Emergency"
                    true
                }
                R.id.nav_needhelp -> {
                    navController.navigate(R.id.navigation_need_help_request)
                    binding.toolbar.title="Requests"
                    true
                }

                else -> {
                    false
                }
            }
        }


    }


}